tests/test_odb.py

```diff
@@ -8,88 +8,89 @@
 from dvc_objects.fs.base import FileSystem
 
 
-def test_odb(memfs):
-    odb = ObjectDB(memfs, "/odb")
+@pytest.fixture
+def odb(memory_path, memfs):
```

Comment:
strictly speaking this (and the rest) should use `tmp_upath` though. Can you get an fs from `tmp_upath`?